.dockerignore

```diff
@@ -0,0 +1,9 @@
+./LICENSE
+./README.md
+devel/
+log/
+build/
+bin/
+lib/
+install/
+*~
```

Comment:
ignore all directories etc that are generated by colcon, also ignore changes to readme etc as they are just documentation